superset/utils/pandas_postprocessing.py

```diff
@@ -599,6 +599,7 @@ def _prophet_fit_and_predict(  # pylint: disable=too-many-arguments
         weekly_seasonality=weekly_seasonality,
         daily_seasonality=daily_seasonality,
     )
+    df["ds"] = df["ds"].dt.tz_convert(None)
```

Comment:
I believe we need to check first if the timezone is set. Can you check to see if this works? (I unfortunately don't have my Druid cluster handy right now)
```suggestion
    if df["ds"].dt.tz:
        df["ds"] = df["ds"].dt.tz_convert(None)
```

Reply:
thank you @villebro. committed your suggestion.